facefusion/choices.py

```diff
@@ -45,7 +45,7 @@
 
 voice_extractor_models : List[VoiceExtractorModel] = [ 'kim_vocal_1', 'kim_vocal_2', 'uvr_mdxnet' ]
 
-workflows : List[WorkFlow] = [ 'auto', 'audio-to-image', 'image-to-image', 'image-to-video' ]
```

Comment:
how about this speaking convention?

workflows: List[WorkFlow] = [
    'auto',
    'audio-to-image:video',
    'image-to-image',
    'image-to-video',
    'image-to-video:frame'
]


the alternative is "as" instead of ":"